openhcl/hcl/src/ioctl.rs

```diff
@@ -1869,14 +1869,14 @@ impl<'a, T: Backing> ProcessorRunner<'a, T> {
     /// Gets the current offload_flags from the run page.
     pub fn get_offload_flags(&self) -> hcl_intr_offload_flags {
         // SAFETY: The `offload_flags` field of the run page will not be concurrently updated.
```

Comment:
Given this, I think you can just have `offload_flags_mut(&mut self) -> &mut hcl_intr_offload_flags`

Reply:
Fixed